[BUG] failing plotting function in annotation module

### Describe the bug

@patrickzib any idea  why this is failing? lists you as author. some change in matplotlib I guess.

https://github.com/aeon-toolkit/aeon/actions/runs/6205841798/job/16849438770?pr=743

```python
plot_time_series_with_change_points
```

### Steps/Code to reproduce the bug

see

https://github.com/aeon-toolkit/aeon/actions/runs/6205841798/job/16849438770?pr=743

### Expected results

pass tests

### Actual results

```python-traceback

in plot_time_series_with_profiles(ts_name, ts, profiles, true_cps, found_cps, score_name, font_size)
    140 for a in ax:
    141     for tick in a.xaxis.get_major_ticks():
--> 142         tick.label.set_fontsize(font_size)
    144     for tick in a.yaxis.get_major_ticks():
    145         tick.label.set_fontsize(font_size)

AttributeError: 'XTick' object has no attribute 'label'

Error: Process completed with exit code 1.


```

### Versions

_No response_

--- a/aeon/annotation/plotting/utils.py
+++ b/aeon/annotation/plotting/utils.py
@@ -139,10 +139,20 @@ def plot_time_series_with_profiles(
 
     for a in ax:
         for tick in a.xaxis.get_major_ticks():
-            tick.label.set_fontsize(font_size)
+            if hasattr(tick, "label"):
+                tick.label.set_fontsize(font_size)
+            if hasattr(tick, "label1"):
+                tick.label1.set_fontsize(font_size)
+            if hasattr(tick, "label2"):
+                tick.label2.set_fontsize(font_size)
 
         for tick in a.yaxis.get_major_ticks():
-            tick.label.set_fontsize(font_size)
+            if hasattr(tick, "label"):
+                tick.label.set_fontsize(font_size)
+            if hasattr(tick, "label1"):
+                tick.label1.set_fontsize(font_size)
+            if hasattr(tick, "label2"):
+                tick.label2.set_fontsize(font_size)
 
     if true_cps is not None:
         for idx, true_cp in enumerate(true_cps):

